### PR TITLE
T37: Create a User Model in the DB Schema

### PR DESCRIPTION
**Pull request by fahadakhan2**
**Description:**
- Added user table to schema.prisma
- ran a prisma migration
- added db to gitignore

### CODE REVIEW

File: backend/prisma/migrations/20240621223936_init_users_table/migration.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+-- CreateTable
```
Comment (justinpchen94) [inline backend/prisma/migrations/20240621223936_init_users_table/migration.sql]: are these autogenerated comments? i don't think you need to keep these

File: backend/server.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+const prisma = require('./utils/prismaClient');
```
Comment (justinpchen94) [inline backend/server.js]: i don't think you need this here yet right? this is just an unused import?

File: backend/prisma/migrations/20240621223936_init_users_table/migration.sql (hunk 1 above)
Comment (fahadakhan2) [reply backend/prisma/migrations/20240621223936_init_users_table/migration.sql]: Yeaup auto generated. I'll go ahead and remove it.

File: backend/server.js (hunk 2 above)
Comment (fahadakhan2) [reply backend/server.js]: Included it there because I thought we'd need it in the future. Removed.

**Review by justinpchen94 — APPROVED** [review]

**Pull request merged**